main/Card/index.tsx

```diff
@@ -21,22 +21,39 @@ const StlyedImage = styled.Image`
   height: 100px;
 `;
 
+const Divider = styled.View`
+  margin: 5px 0px;
+  height: 0.7px;
+  background-color: lightgray;
+`;
+
 interface Props {
   testID?: string;
   containerStyle?: ViewStyle;
   children?: ReactNode | ReactNodeArray;
   image?: ImageSourcePropType;
   imageStyle?: ImageStyle;
   contentsStyle?: ViewStyle;
+  divider?: boolean;
+  dividerStyle?: ViewStyle;
 }
 
 const Card: FC<Props> = (props) => {
-  const { containerStyle, children, image, imageStyle, contentsStyle } = props;
+  const {
+    containerStyle,
+    children,
+    image,
+    imageStyle,
+    contentsStyle,
+    divider,
+    dividerStyle,
+  } = props;
   return (
     <Container style={[containerStyle]}>
       {image && <StlyedImage source={image} style={[imageStyle]} />}
       {children && (
         <ContentsContainer style={[contentsStyle]}>
+          {divider && <Divider style={[dividerStyle]} />}
```

Comment:
```suggestion
          {title !== undefined && children && <Divider style={[dividerStyle]} />}
```
i think it's better to replace divider to title && children

Reply:
I changed